mis_builder_budget_operating_unit/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+=================================
+Mis Builder Budget Operating Unit
+=================================
+
+Introduces the operating unit to MIS Builder Budget instances.
+
+Usage
+=====
+
+Set the operating unit in an MIS Builder budget instance.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/213/10.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/operating-unit/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smash it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Author
+------
+
+* Artem Kostyuk <a.kostyuk@mobilunity.com>
+
+Maintainer
+----------
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```

Comment:
For new modules, I suggest using the new README fragments: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst